### PR TITLE
Add positive test, update `validVc.json` and fix failing tests

### PR DESCRIPTION
**Pull request by JSAssassin**

### CODE REVIEW

File: tests/10-issuer.js [hunk 1]
```diff
@@ -31,12 +31,38 @@ describe('Issue Credential - Data Integrity', function() {
     const issuer = implementation.issuers.find(
       issuer => issuer.tags.has('VC-API'));
     describe(name, function() {
+      it('MUST successfully issue a credential.', async function() {
```
Comment (aljones15) [inline tests/10-issuer.js:34]: good and thanks for adding this.

File: tests/assertions.js [hunk 2]
```diff
@@ -12,6 +12,26 @@ function shouldThrowInvalidInput({result, error}) {
     'Expected status code 400 invalid input!');
 }
 
+function testIssuedVc({issuedVc}) {
+  issuedVc.should.be.an('object');
+  issuedVc.should.have.property('@context');
+  issuedVc.should.have.property('type');
+  issuedVc.type.should.eql(['VerifiableCredential']);
```
Comment (aljones15) [inline tests/assertions.js:19]: this is ok, but Vcs usually have multiple types. might be best to assume one type is `VerifiableCredential`.

**Review by aljones15 — APPROVED** [review]
LGTM, feel free to merge when you're comfortable.

File: tests/assertions.js (hunk 2 above)
Comment (JSAssassin) [reply tests/assertions.js]: Updated [15cd05d](https://github.com/w3c-ccg/vc-api-issuer-test-suite/pull/7/commits/15cd05d2f671f9b456ac3435244b19100499de51)

File: tests/10-issuer.js [hunk 3]
```diff
@@ -101,50 +126,50 @@ describe('Issue Credential - Data Integrity', function() {
...
+        const credential = createValidVc({issuerId});
+        const invalidCredentialTypes = [null, true, 4, []];
+        for(const invalidCredentialType of invalidCredentialTypes) {
+          credential.type = invalidCredentialType;
+          const body = {credential};
+          const {result, error} = await issuer.issue({body});
+          shouldThrowInvalidInput({result, error});
+        }
       });
       it('credential MUST have property "issuer"', async function() {
```
Comment (JSAssassin) [inline tests/10-issuer.js:139]: @aljones15 DB is failing this test now, instead of throwing error we are getting a 201 response back.

Comment (aljones15) [reply tests/10-issuer.js:139]: yeah this known. We add an issuer if one is not there. The best course of action I can think of is to just to wait and watch, if you're not subscribed to the VC-API, subscribe.

**Pull request merged**